src/core/providers/ApiProvider.tsx

```diff
@@ -160,7 +160,7 @@ export const ApiProvider = ({ children }: PropsWithChildren) => {
     };
 
     // Retrieve existing token from SecureStore, if any
-    Keychain.getGenericPassword()
+    Keychain.getGenericPassword(kcSettings)
```

Comment:
could be worth making a credentials getter in authHooks, so that there are no other references of they keychain around the codebase and we can avoid exposing the kcSettings object...